packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -183,6 +187,14 @@ export class Cascader extends React.PureComponent<CascaderProps, CascaderState>
     });
   };
 
+  onSelectInputChange = (value: string) => {
```

Comment:
This is here so that if I start searching and then delete all the text it goes back to standard cascade UI. Before it was "stuck" in the select/search mode.